src/settings/InventoryRecordTypeSelectField.js

```diff
@@ -3,16 +3,27 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 
 import { FieldSelect, FieldSelectFinal } from '@folio/stripes-acq-components';
-import { INVENTORY_RECORDS_TYPE_FOR_SELECT } from '../components/POLine/const';
+import {
+  INVENTORY_RECORDS_TYPE_FOR_SELECT,
+  OPTION_WITH_BINDARY_ACTIVE,
+} from '../components/POLine/const';
 
-const InventoryRecordTypeSelectField = ({ label, isRedux, ...rest }) => {
+const InventoryRecordTypeSelectField = ({
+  label,
+  disabled,
+  isRedux,
+  isBindaryActive = false,
+  ...rest
+}) => {
   const FieldComponent = isRedux ? FieldSelect : FieldSelectFinal;
+  const dataOptions = isBindaryActive ? OPTION_WITH_BINDARY_ACTIVE : INVENTORY_RECORDS_TYPE_FOR_SELECT;
```

Comment:
What would a user see as the selected option if `instance`, `instanceAndHolding`, or `none` was selected before activating `isBindaryActive`?

Reply:
That what I meant:

https://github.com/folio-org/ui-orders/assets/88109087/8f947c93-c4f1-4806-b3ba-ee803b0ee99c

Reply:
Thank you for the detailed review.


https://github.com/folio-org/ui-orders/assets/116072773/59aad65e-fc34-459e-8503-d2ee50c651d7